<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphModifyWidget</class>
 <widget class="QDialog" name="GraphModifyWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Graph</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" rowspan="5" colspan="2">
    <widget class="QListWidget" name="list2">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="renameButton">
     <property name="text">
      <string>Rename..</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5" colspan="2">
    <widget class="QListWidget" name="list1">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="unlinkButton">
     <property name="text">
      <string>Unlink</string>
     </property>
     <property name="icon">
      <iconset resource="topo-blend.qrc">
       <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="linkButton">
     <property name="text">
      <string>Link</string>
     </property>
     <property name="icon">
      <iconset resource="topo-blend.qrc">
       <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="unlinkAllButton">
     <property name="text">
      <string>Unlink all</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="updateLinkButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDoubleSpinBox" name="uA">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="vA">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QDoubleSpinBox" name="vB">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove node</string>
     </property>
     <property name="icon">
      <iconset resource="topo-blend.qrc">
       <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QDoubleSpinBox" name="uB">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="topo-blend.qrc"/>
 </resources>
 <connections/>
</ui>
